medusa/server/web/home/add_shows.py

```diff
@@ -308,7 +310,7 @@ def popularShows(self):
 
         try:
             recommended_shows = ImdbPopular().fetch_popular_shows()
-        except (StandardError, requests.RequestException) as e:
+        except (RequestException, socket.gaierror, StandardError) as e:
```

Comment:
As explained. The requests lib, swallows the socket.gaierror through `except socket.error`.
https://github.com/kennethreitz/requests/blob/master/requests/adapters.py#L485